specs/provisioningInfo.md

```diff
@@ -0,0 +1,94 @@
+# Provisioning Info for Permissions in Graph
+
+## Abstract
+The provisioning info file is a complementary JSON file to the JSON permission files that contains a list of permissions and the cloud environments in which they are present. It can serve to instruct publishing on which environments to publish a new permission to as well as show which clouds existing permissions are present in.
```

Comment:
Could we merge this with the provisioning info entries already in the main file? https://github.com/microsoftgraph/msgraph-metadata/blob/dm/permissionspec/specs/permissions.md#provisioning-info-object